src/API/Reports/Categories/DataStore.php

```diff
@@ -68,10 +68,8 @@ class DataStore extends ReportsDataStore implements DataStoreInterface {
 	 * Constructor
 	 */
 	public function __construct() {
-		global $wpdb;
-		$table_name = $wpdb->prefix . self::TABLE_NAME;
-		// Avoid ambigious column order_id in SQL query.
-		$this->report_columns['orders_count'] = str_replace( 'order_id', $table_name . '.order_id', $this->report_columns['orders_count'] );
+		$this->set_db_table_name();
```

Comment:
> What do you think about putting this in the parent constructor

I started with that and found that with `const` properties, the value used is the one from the class that the executing code is in (PHP 7.3). If `$table_name` was changed to a `private` var then it could be done in the parent constructor.

Reply:
That might be reason enough to change the type of variable - the only place these constants are accessed outside of `self` is in some test code. With an access change to `get_db_table_name()` we could refactor.

Reply:
Done in https://github.com/woocommerce/woocommerce-admin/pull/2961/commits/dea5454f871265b9e70eee8007476cd33a48f503